nuttx/arch/arm/include/stm32/chip.h

```diff
@@ -493,1571 +208,125 @@
 #  define STM32_NI2C                     3   /* I2C1-3 */
 #  define STM32_NCAN                     1   /* bxCAN */
 #  define STM32_NSDIO                    1   /* SDIO */
-#  define STM32_NLCD                     1   /* LCD 4x44, 8x40*/
+#  define STM32_NLCD                     1   /* LCD 4x32, 8x28*/
 #  define STM32_NUSBOTG                  1   /* USB OTG FS (only) */
-#  define STM32_NGPIO                    82  /* GPIOA-H */
+#  define STM32_NGPIO                    57  /* GPIOA-C, plus few in GPIOD,G, H */
```

Comment:
OK, should revert then back to original chip.h file.   I will rebase back to that file and test again.